### PR TITLE
Build and test on different Java versions

### PR DESCRIPTION
**Pull request by arminru**

### CODE REVIEW

File: gradlew.bat [hunk 1]
```diff
@@ -64,28 +64,14 @@ echo location of your Java installation.
...
-:win9xME_args
-@rem Slurp the command line arguments.
-set CMD_LINE_ARGS=
-set _SKIP=2
-
-:win9xME_args_slurp
-if "x%~1" == "x" goto execute
-
-set CMD_LINE_ARGS=%*
-
```
Comment (pirgeo) [inline gradlew.bat:81]: are these not needed any more?

File: gradlew.bat [hunk 2]
```diff
@@ -64,28 +64,14 @@ echo location of your Java installation.
...
 :execute
 @rem Setup the command line
 
 set CLASSPATH=%APP_HOME%\gradle\wrapper\gradle-wrapper.jar
 
+
 @rem Execute Gradle
-"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %JAVA_OPTS% %GRADLE_OPTS% "-Dorg.gradle.appname=%APP_BASE_NAME%" -classpath "%CLASSPATH%" org.gradle.wrapper.GradleWrapperMain %CMD_LINE_ARGS%
+"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %JAVA_OPTS% %GRADLE_OPTS% "-Dorg.gradle.appname=%APP_BASE_NAME%" -classpath "%CLASSPATH%" org.gradle.wrapper.GradleWrapperMain %*
```
Comment (pirgeo) [inline gradlew.bat:74]: does this `*` replace the command line args from above?

**Review by pirgeo — COMMENTED** [review]
were these changes introduced by updating gradle to a newer version?

**Review by danielkhan — APPROVED** [review]

File: gradlew.bat (hunk 1 above)
Comment (arminru) [reply gradlew.bat:81]: That's updated automatically when upgrading the wrapper:
https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper

**Review by pirgeo — APPROVED** [review]

**Pull request merged**